### ISSUE TITLE
Can LIST_FILES accept a VAR?

### ISSUE THREAD

**Issue by BillyMcSkintos** [electroly/sqlnotebook#54]
Good morning,

Can your LIST_FILES function accept a VAR for the root_path e.g. a folder string?

**Comment by electroly** [comment]
Yes, `LIST_FILES` can accept a variable for the `root_path` parameter.

![Image](https://github.com/user-attachments/assets/9a1d24bb-8831-490b-a283-089820a6cd5c)
